tasks/main.yml

```diff
@@ -907,6 +907,16 @@
           no_log: true
           changed_when: true
 
+        - name: Ensure proper configuration on user {{ mssql_ad_sql_user }}
+          command: >-
+            adutil user modify
+            --name {{ mssql_ad_sql_user }}
+            --distname {{ mssql_ad_sql_user_dn | quote }}
+            --enableaessupport
+            --debug
+          when: __mssql_adutil_account_exists.stdout | bool
```

Comment:
Does `__mssql_adutil_account_exists.stdout | bool` evaluate to `false` if stdout is empty, or `true` otherwise?

Reply:
Command `adutil account exists --name sqluser --accept-eula` returns `true` or `false`